### PR TITLE
[Alerting]: Fix empty rules evaluation statuses

### PR DESCRIPTION
**Pull request by papagian**
**What this PR does / why we need it**:

[GET /api/prometheus/{Recipient}/api/v1/rules](https://grafana.github.io/alerting-api/#/prometheus/RouteGetRuleStatuses
) used to returned an empty result because `GetRuleGroupAlertRules()` requires an non empty namespaceUID

**Updated:**
It also returns the namespace in the unused field `File` (required by the frontend)

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/services/ngalert/api/api_prometheus.go [hunk 1]
```diff
@@ -63,8 +63,12 @@ func (srv PrometheusSrv) RouteGetRuleStatuses(c *models.ReqContext) response.Res
 		return response.JSON(http.StatusInternalServerError, ruleResponse)
 	}
 
-	for _, groupId := range ruleGroupQuery.Result {
-		alertRuleQuery := ngmodels.ListRuleGroupAlertRulesQuery{OrgID: c.SignedInUser.OrgId, RuleGroup: groupId}
+	for _, r := range ruleGroupQuery.Result {
+		if len(r) < 3 {
```
Comment (davidmparrott) [inline pkg/services/ngalert/api/api_prometheus.go:67]: @papagian is it possible to ever have this be true? All three fields are not nullable so shouldn't we always have all three fields set in any result?

**Review by davidmparrott — APPROVED** [review]
LGTM 🎉

File: pkg/services/ngalert/api/api_prometheus.go (hunk 1 above)
Comment (papagian) [reply pkg/services/ngalert/api/api_prometheus.go:67]: I think it's not possible but I've added just to be safe. Do you think it's better to remove it?

Comment (davidmparrott) [reply pkg/services/ngalert/api/api_prometheus.go:67]: No I think it's fine, just checking that I understand what's going on correctly. The check is sensible and about as light weight as possible.

**Pull request merged**